File: MultiScreenApp/app/src/main/java/com/example/multiscreentest/MainActivity.java
package com.example.multiscreentest;

import androidx.appcompat.app.AppCompatActivity;

import android.content.Intent;
import android.os.Bundle;
import android.util.Log;
import android.view.View;
import android.widget.TextView;

import java.util.ArrayList;

public class MainActivity extends AppCompatActivity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);
    }

    public void openNumberList(View view) {

        Intent intent = new Intent(this, NumbersActivity.class);
        startActivity(intent);
    }

    public void openFamilyMembers(View view) {
        Intent intent = new Intent(this, FamilyMembersActivity.class);
        startActivity(intent);
    }

    public void openColor(View view) {
        Intent intent = new Intent(this, ColorsActivity.class);
        startActivity(intent);
    }

    public void openPhrases(View view) {
        Intent intent = new Intent(this, PhrasesActivity.class);
        startActivity(intent);
    }
}

File: MultiScreenApp/app/src/main/java/com/example/multiscreentest/MainActivity.kt
package com.example.multiscreentest

import android.content.Intent
import android.os.Bundle
import android.view.View
import androidx.appcompat.app.AppCompatActivity
import com.example.multiscreentest.ui.colors.ColorsActivity
import com.example.multiscreentest.ui.family.FamilyMembersActivity
import com.example.multiscreentest.ui.numbers.NumbersActivity
import com.example.multiscreentest.ui.phrase.PhrasesActivity

class MainActivity : AppCompatActivity() {

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)
    }

    fun openNumberList(view: View?) {
        val intent = Intent(this, NumbersActivity::class.java)
        startActivity(intent)
    }

    fun openFamilyMembers(view: View?) {
        val intent = Intent(this, FamilyMembersActivity::class.java)
        startActivity(intent)
    }

    fun openColor(view: View?) {
        val intent = Intent(this, ColorsActivity::class.java)
        startActivity(intent)
    }

    fun openPhrases(view: View?) {
        val intent = Intent(this, PhrasesActivity::class.java)
        startActivity(intent)
    }
}